bitsail-connectors/connector-druid/src/main/resources/druid-type-converter.yaml

```diff
@@ -0,0 +1,48 @@
+engine.type.to.bitsail.type.converter:
```

Comment:
I believe this should be bitsail type to engine type for the sink connector

Reply:
The usage of [FileMappingTyhpeInfoReader](https://github.com/bytedance/bitsail/blob/master/bitsail-common/src/main/java/com/bytedance/bitsail/common/type/filemapping/FileMappingTypeInfoReader.java) (the class to parse converter YAML file) may be misleading, but how it actually works with [DelegateFlinkWriter](https://github.com/bytedance/bitsail/blob/master/bitsail-cores/bitsail-core-flink/src/main/java/com/bytedance/bitsail/flink/core/delagate/writer/DelegateFlinkWriter.java) is by using `engine.type.to.bitsail.type.converter` to convert from BitSailConfiguration to TypeInfo. That's why we see a few Sink connector only has `engine.type.to.bitsail.type.converter` but still work properly. On the contrary, I'm not quite sure how `bitsail.type.to.engine.type.converter` is used but I can see inconsistency in the system. A few refactoring changes may be needed to justify or merge usage of those two converter methods.

Reply:
@hk-lrzy do you have a quick answer for this question? I am a bit uncertain as well.